web/src/containers/ExecutiveSummary.js

```diff
@@ -1,14 +1,15 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { connect } from 'react-redux';
+import PropTypes from 'prop-types'
```

Comment:
Different prettier config?  All the semicolons are gone.  😬

Reply:
oops, fixing

Reply:
fixed